composer.json

```diff
@@ -19,7 +19,7 @@
 	    "php": ">=7.1.0",
         "phpunit/phpunit": ">=7.0.0",
         "mikey179/vfsStream": "1.*",
-        "allure-framework/allure-php-api": "~1.1.0"
+        "allure-framework/allure-php-api": "~1.2.0"
```

Comment:
Should it rather be `~1.1` so that allure-phpunit does not demand higher version of dependency and thus, probably, higher subdependencies which may in turn add incompatibility issues for someone?